File: Biblioteca/API/Models/Emprestimo.cs
public class Emprestimo
{
    public String Id { get; set; } = Guid.NewGuid().ToString();
    required public String LivroId { get; set; }
    public Livro? Livro { get; set; }
    required public String LeitorId { get; set; }
    public Leitor? Leitor { get; set; }
    public DateTime DataEmprestimo { get; set; } = DateTime.Now;
    public DateTime? DataDevolucao { get; set; }
}

File: Biblioteca/front/src/models/Emprestimo.ts
import Leitor from "./Leitor";
import Livro from "./Livro";

export default interface Emprestimo {
    id?: string;
    livroId: string;
    livro?: Livro;
    leitorId: string;
    leitor?: Leitor;
    dataEmprestimo?: string;
    dataDevolucao?: string;
}

export interface EmprestimoListagem {
    id_Emprestimo: string;
    titulo_Livro: string;
    nome_Leitor: string;
    data_Emprestimo: string;
}
